resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -7,4 +7,4 @@ if [ ! -e "$JRUBY_JAR" ]; then
   return 1
 fi
 
-CLASSPATH="${CLASSPATH}:${JRUBY_JAR}"
+CLASSPATH="${CLASSPATH}:${JRUBY_JAR}:/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/jars/*"
```

Comment:
The dir is created with a `700` permission, does that seem right? https://github.com/puppetlabs/ezbake/blob/fe21baaac64457935c540f3907815e1f57c51c46/resources/puppetlabs/lein-ezbake/template/global/install.sh.erb#L190-L192